pandas/tests/frame/test_query_eval.py

```diff
@@ -203,6 +203,15 @@ def test_eval_simple(self, engine, parser):
         tm.assert_series_equal(expected, res)
 
 
+class TestDataFrameEvalWithMultiIndex:
```

Comment:
This test doesn't use a MultiIndex. I'd keep it part of `TestDataFrameEval`

Reply:
Okay works